scalding/src/main/scala/com/matthewrathbone/scalding/Main.scala

```diff
@@ -0,0 +1,35 @@
+package main.scala.com.matthewrathbone.scalding
+
+import com.twitter.scalding._
+import cascading.tuple.TupleEntry
+
+class Main ( args: Args ) extends Job( args )
+  {
+  
+    val inputFields = 'line
+	val users = ( 'id, 'email, 'language, 'location)
+  	val transactions = ( 'transaction_id, 'product_id, 'user_id, 'purchase_amount, 'item_description);
+	
+	val input1 = TextLine( args( "input1" ) )
```

Comment:
Hi Matthew, yes, I have added transactions_test.txt and users_test.txt to the scalding folder, in the same way as they live in the cascading folder, without data/ .